monkeytype/tracing.py

```diff
@@ -161,6 +165,18 @@ def get_func(frame: FrameType) -> Optional[Callable]:
     return func
 
 
+def get_caller_func_info(frame: FrameType) -> Optional[str]:
+    """Return the information of function who calls func who has code object."""
+    caller_frame = frame.f_back
+    if caller_frame is None:
+        return None
```

Comment:
It seems unnecessary to have these lines duplicated both here and in `_get_caller_func_info` method below. I would instead have this function be called `get_frame_location_info` and have it take the caller frame instead of the called frame, i.e. just eliminate these three lines from it.